scanner/src/main/kotlin/ScanResultsStorage.kt

```diff
@@ -75,6 +79,35 @@ abstract class ScanResultsStorage {
 
             log.info { "Using Artifactory storage at ${config.url}." }
         }
+
+        /**
+         * Configure a [PostgresStorage] as the current storage backend.
+         */
+        fun configure(config: PostgresStorageConfiguration) {
+            require(config.url.isNotBlank()) {
+                "URL for PostgreSQL storage is missing."
+            }
+
+            require(config.schema.isNotBlank()) {
+                "Database for PostgreSQL storage is missing."
+            }
+
+            require(config.username.isNotBlank()) {
+                "Username for PostgreSQL storage is missing."
+            }
+
+            require(config.password.isNotBlank()) {
+                "Password for PostgreSQL storage is missing."
+            }
+
+            val properties = Properties()
+            properties["user"] = config.username
```

Comment:
Nit: Maybe use `also` to set these?

Reply:
I think in this case it's not improving readability.